guide/getting-started/technology-primer.md

```diff
@@ -73,264 +79,265 @@ It's both. Lowercase bitcoin refers to the currency. So you may have 1 bitcoin i
+   alt-text = "Transactions moving from the mempool into a block"
    width = 400
    height = 400
    layout = "float-right-desktop"
 %}
 
-A private key is used to scramble information. The matching public key (which we call an address in Bitcoin) can unscramble the information again. This allows for the transfer of information without any third party being able to understand it.
+While miners check that the transactions included in the block are valid, they must also be kept accountable that the information they are submitting is correct. While checking transactions for a new block, miners must also race to solve a cryptographic puzzle as proof that they have correctly done the verification. This process is called Proof of Work, and it is fundamental to the security of the blockchain. As compensation, the first miner to submit a valid block to the network, solving the puzzle, will earn all the fees from the transactions in the block and a block reward.
 
-Note that there is a strong element of identity in this mechanism, as it ensures that only the recipient can unscramble the message while also verifying that the message came from the expected sender. This requires that both parties securely exchange public keys with each other in a secure way.
+We also call the Bitcoin blockchain "base layer" due to the strong security it provides.
 
 **More info**
-- [Public key cryptography](https://en.wikipedia.org/wiki/Public-key_cryptography){:target="_blank"}
-
+- Mining [software]({{ '/guide/getting-started/software/#mining-software' | relative_url }}) and [hardware]({{ '/guide/getting-started/hardware/#mining-hardware' | relative_url }})
 </div>
 
-## Signature
+## Do all transactions have to be this secure?
 
 <div class="center" markdown="1">
 
 {% include image.html
-   image = "/assets/images/guide/getting-started/technology/signature.jpg"
-   retina = "/assets/images/guide/getting-started/technology/signature@2x.jpg"
-   alt-text = "Illustration of a transaction signature"
+   image = "/assets/images/guide/getting-started/technology/layers.jpg"
+   retina = "/assets/images/guide/getting-started/technology/layers@2x.jpg"
+   alt-text = "Other networks connected to the Bitcoin blockchain"
    width = 400
    height = 400
    layout = "float-right-desktop"
 %}
 
-Since a private key can be used to prove that the holder controls a specific address, it can therefore authorize transactions from the address. This is called a digital signature.
-
-One of the most important activities of the Bitcoin network is to verify that signatures are valid.
+Layers in Bitcoin serve to provide alternative settlement rules. Bitcoin's block generation time of 10 minutes and the 1 MB size means that the blockchain can handle around 300 000 transactions per day or 4.6 transactions per second. Layer 1, the base layer, is the most secure, so it has the most time-consuming verification process.
```

Comment:
Not sure how to understand the words `Layers in Bitcoin`. Here is some alternative phrasing for this section I had noted in Figma:

> The Bitcoin network is optimized for decentralization and security and can process around 300 000 transactions per day, or 4.6 transactions per second.
> 
> This is not enough to scale as a payment method of the internet and meet the demands of our global economy. To make this possible, additional layers with different properties can be built on Bitcoin.
> 
> The Lightning network is such a second layer and allows for fast and frequent payments.